S3/S3.py

```diff
@@ -707,7 +711,7 @@ def object_modify(self, src_uri, dst_uri, extra_headers = None):
         headers = self._sanitize_headers(headers)
         acl = self.get_acl(src_uri)
 
-        headers['x-amz-copy-source'] = "/%s/%s" % (src_uri.bucket(), self.urlencode_string(src_uri.object()))
+        headers['x-amz-copy-source'] = deunicodise("/%s/%s" % (src_uri.bucket(), self.urlencode_string(src_uri.object())), encoding="UTF-8")
```

Comment:
why use deunicodise() here instead of .encode('UTF-8') as you did with the same header at line 682?